.gitlab/.pre/gitlab_configuration.yml

```diff
@@ -62,6 +62,8 @@ lint_gitlab_ci_jobs_needs_rules:
     - changes:
         paths:
           - .gitlab/.ci-linters.yml
+          - .gitlab/JOBOWNERS
```

Comment:
Do we really need that? I think this is already included in `.on_gitlab_changes` isn'it?

Reply:
Indeed, nice catch